docs/source/guides/speed.rst

```diff
@@ -199,12 +197,25 @@ Read more in our :ref:`accelerators` and :ref:`plugins` guides.
 
 -----------
 
+**************
+Early Stopping
+**************
+
+Usually long training epochs can lead to either overfitting or no major improvements in your metrics due to no limited convergence. Here Early Stopping can help you stop
+the training entirely by monitoring a metric of your choice. To configure this you can use :class:`~pytorch_lightning.callbacks.early_stopping.EarlyStopping`.
+You can read more about it :ref:`here <early_stopping>`.
+
+----------
+
 .. _speed_amp:
 
 *********************************
-Mixed precision (16-bit) training
+Mixed Precision (16-bit) Training
 *********************************
 
+Lower precision, such as the 16-bit floating-point, enables the training and deployment of marge neural networks since they require less memory, enhances data transfer operations since they required
```

Comment:
```suggestion
Lower precision, such as the 16-bit floating-point, enables the training and deployment of large neural networks since they require less memory, enhance data transfer operations since they required
```